### ISSUE TITLE
Debian: unable to start services after merge 

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#5481]
**Describe the bug**
After merge of PR #5392, PF is not able to start on Debian with following errors:
```
FATAL - /usr/local/pf/logs/packetfence.log must be owned by user pf. Fix with chown pf -R /usr/local/pf/logs/
FATAL - /usr/local/pf/logs/pfconfig.log must be owned by user pf. Fix with chown pf -R /usr/local/pf/logs/
```
A `pfcmd fixpermissions` has been done before starting `pf` services.

No issue on CentOS.

**Additional context**
See https://gitlab.com/inverse-inc/packetfence/-/jobs/570570936#L374

**Comment by julsemaan** [comment]
I didn't change anything to the packaging in this PR.

Are you really sure this is caused by the log tailer PR ?

**Comment by nqb** [comment]
Hi @julsemaan,

I assumed this is caused by #5392 and the OS detection stuff. Pipeline passed yesterday without issue before merge of #5459 and #5392.

**Comment by nqb** [comment]
Current tree on Debian:
```
root@pfdeb9dev:/usr/local/pf/logs# tree 
.
|-- packetfence.log
`-- usr
    `-- local
        `-- pf
            `-- logs
                |-- api-frontend.log
                |-- mariadb_error.log
                |-- packetfence.log
                |-- pfconfig.log
                |-- redis_cache.log
                `-- redis_server.log
```

**Comment by julsemaan** [comment]
it seems this could be caused by a faulty rsyslog configuration

I fixed the rsyslog config in cee1cc6f37d6ceb3a092ad99794cec6666a1d067

Can you see if that helps this too?

**Comment by nqb** [comment]
Looks good according to https://gitlab.com/inverse-inc/packetfence/-/jobs/572398293
